### ISSUE TITLE
Why did the version get bumped to 3.0.41 in the podspec, but not GitHub?

### ISSUE THREAD

**Issue by tunabelly** [PaddleHQ/Mac-Framework#229]
Just noticed that the podspec got updated with a version of 3.0.41 for the v3 framework, but there's no new release to go with it. Was this intentional?

<img width="1019" alt="Screen Shot 2019-03-15 at 8 34 14 PM" src="https://user-images.githubusercontent.com/2223932/54468338-e885e680-4761-11e9-9e1b-41bbd585a5bb.png">

**Comment by GregularExpressions** [comment]
@tunabelly Yes this was deliberate, it was to mark the v3 pod as deprecated in favour of v4.

**Comment by tunabelly** [comment]
@GregularExpressions ok thanks for clearing that up.
